README.md

```diff
@@ -170,3 +165,17 @@ http://localhost/frontend
 and geoserver at:
 
 http://localhost/geoserver
+
+
+## How to update the SCIP demo instance
+
+This application has not yet had a full release, and has no production instances, but there is a semipermanent demo instance running on docker-dev02. You will not need to worry about CORS or proxies when working with this instance, as a proxy is already set up for it.
+
+1. Clone this repository to anywhere on docker-dev02 and check out whatever branch you want to demo 
+2. Build the docker image with `docker build -t scip:your-branch-name -f docker/Dockerfile .`
+3. Make a new dated directory in /storage/data/projects/comp_support/bc-srif/dev-deployment/ for your demo and copy files into it from the most recent existing directory (note these are not the same as the docker-compose files files in this repository)
```

Comment:
At some point, the sooner the better, the dev directory name should be changed from `.../dev-deployment` to just `.../dev`, to conform with the standard we are/will be using for all the other apps (some existing apps still use their old, idiosyncratic directory names).

Prod deployment directory is, naturally, `.../prod`

Reply:
I've moved the deployment artifacts into that directory and will update the readme, though I have not moved the geoserver data mount, which I think might break if I move it.

Reply:
Fair enough. Don't mess with delicate flowers!